src/main/java/chanceCubes/rewards/giantRewards/ThrowablesReward.java

```diff
@@ -45,9 +44,9 @@ public void update()
 				else if(entChoice == 1)
 				{
 					throwEnt = EntityType.FIREBALL.create(level);
-					((Fireball) throwEnt).xPower = 0.1f * (-1 + (Math.random() * 2));
-					((Fireball) throwEnt).yPower = 0.1f * (-1 + (Math.random() * 2));
-					((Fireball) throwEnt).zPower = 0.1f * (-1 + (Math.random() * 2));
+//					((Fireball) throwEnt).xPower = 0.1f * (-1 + (Math.random() * 2));
```

Comment:
so the fireballs won't move?

Reply:
the power variables are POOF....

Reply:
From testing the fireballs still shoot out fine